Modules/Core/Common/test/itkImageTest.cxx

```diff
@@ -61,15 +61,12 @@ itkImageTest(int, char *[])
   image->GetSource();
   image->DisconnectPipeline();
 
-  auto                 spacing = itk::MakeFilled<Image::SpacingType>(1.0);
-  auto                 origin = itk::MakeFilled<Image::PointType>(1.0);
+
   Image::DirectionType direction;
   direction[0][0] = .5;
   direction[0][1] = .7;
   direction[1][0] = .7;
   direction[1][1] = .5;
-  image->SetSpacing(spacing);
-  image->SetOrigin(origin);
```

Comment:
@hjmjohnson  This is a behavior change! In this particular case, `image->SetOrigin(origin)` set the origin to `{1.0, 1.0}`, originally. Was it your intention to change it to the default origin `{0.0, 0.0}` ?

Reply:
This was a BAD test.  The value of origin had no-effect on what is tested.    The test does not change what it checks no-matter what the value of origin is.  It only exercised SetOrigin, but did not verify that the behavior was correct.

The testing of SetOrigin with values like "1234.0" and verifying that the behavior is correct is done elsewhere.

With regards to this test, the "SetOrigin" is pointless because it is untested and unused.

Reply:
OK, thanks Hans, I agree, a unit test should not just "exercise" a function, it should also check the effect of such a function call. Otherwise I agree that the function call is pointless.